src/app/profile/tenant/tenant.component.html

```diff
@@ -0,0 +1,81 @@
+<div id="overview" class="container-fluid content padding-top-15">
+  <div class="row profile-edit">
+    <form role="form" #tenantForm="ngForm">
+      <!-- left column -->
```

Comment:
Why is the left column 5 offset by 1, but the right column is 4, offset by 1? so the right column is one less column wide, and is pushed a full column away from the left one? These also do not equal 12, which should always be the case.